clients/src/test/java/org/apache/kafka/common/metrics/MetricsTest.java

```diff
@@ -377,10 +377,20 @@ public void testSampledStatInitialValue() {
         assertEquals(0.0, sampledTotal.measure(config, time.milliseconds()), EPS);
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
```

Comment:
if you choose to make this controllable, add a test where this explodes

Reply:
Added.